### DISCUSSION TITLE
Changing the flow doesn't save between restarts. Is this intended?

### DISCUSSION THREAD

**Discussion by alexrmay91** [Jyers/Marlin#756] [Q&A] [answered]
Whenever I set flow (in the menu, not during a print), I store settings. After turning off my printer, the flow settings go back to 100%. Not sure if this is intentional or possible. But I'd love to see this setting stick between restarts if possible.

**Comment by Cal-LF** [comment]
No thanks. Set the flow in the slicer if you want it permanent.

**Comment by Jyers** [comment, marked as answer]
This is just how marlin handles it, I suppose it's meant to be on the fly compensation not a persistent change
